R/misc.R

```diff
@@ -180,14 +186,13 @@ add_Y_residuals <- function(data_list, conditions, y_as_matrix = FALSE, scale_re
                      unnest(Y_resid) %>%
                      as.matrix()
     colnames(Y_resid_matrix) <- conditions
-    data_list$Y_resid <- Y_resid_matrix
+    data_list$Y_resid <- list(Y_resid_matrix)
```

Comment:
Please revert this line for now. I believe what we previously had is correct but i'm yet to test that --- it is going to be relevant to the mvSuSiE pipeline which exists but have not yet been tested to work with pecotmr. You can revert it for now. My team will take care of testing it.